roles/pulp/tasks/install.yml

```diff
@@ -65,10 +65,35 @@
       register: result
       until: result is succeeded
 
+    - name: Install RPM prerequisities
+      package:
+        name: '{{ item }}'
+        state: present
+      with_items:
+        - createrepo_c
+        - python3-createrepo_c
```

Comment:
This only exists in Fedora, not in CentOS 7, because python3 is not formally in CentOS 7.

We could use copr & software collections to provide it for CentOS 7 on top of a Python 3.x software collection. Or maybe just modify the package to build in a non-standard way.

Reply:
This is what you see on CentOS7:
```
TASK [pulp : Install RPM prerequisities] ***************************************
changed: [pulp3-source-centos7] => (item=createrepo_c)
failed: [pulp3-source-centos7] (item=python3-createrepo_c) => {
    "changed": false,
    "item": "python3-createrepo_c",
    "rc": 126,
    "results": [
        "No package matching 'python3-createrepo_c' found available, installed or updated"
    ]
}

MSG:

No package matching 'python3-createrepo_c' found available, installed or updated
```
I did this using pulplift & pulp3-source-centos7

Reply:
I think we should close this PR and use the instructions [from here](https://pulp-rpm.readthedocs.io/en/latest/quickstart.html) which use fedora29 until we can ship pulp_rpm as a container itself.

Reply:
Also, bmbouter was under the impression that python3-createrepo_c exits for EPEL7 due to this page:
https://apps.fedoraproject.org/packages/python3-createrepo_c

It never has. Even version 0.10.0-6.el7 of createrepo_c never built the python3-createrepo_c subpackage for epel7 (you'll see this under the bodhi link.) The page is mistaken.